### PR TITLE
Big fat E.G.O. dump

### PR DESCRIPTION
**Pull request by Coxswain-Navigator**
## About The Pull Request
Adds E.G.O. for My sweet orange tree and Someone's Portrait, adds E.G.O. gifts for pale horse, Puss in Boots, and blubbering toad.
All My sweet orange tree E.G.O. and blubbering toad armor E.G.O. by Rotipik#8251
Someone's portrait E.G.O., toad weapon, and all gifts by myself.
Adds Wishing Cairn by Mr. heavenly, weapon sprite by loveGreenstuff

## Why It's Good For The Game
Makes the eventual merging process a lot easier.

## Changelog
:cl:
add: Added a whole bunch of E.G.O. to the code.
/:cl:

### CODE REVIEW

**Comment by Coxswain-Navigator** [comment]
Innocence needs icons before this gets merged.

File: code/modules/clothing/suits/ego_gear/waw.dm [hunk 1]
```diff
@@ -342,3 +342,12 @@
 	attribute_requirements = list(
 					PRUDENCE_ATTRIBUTE = 80
 					)
+
+/obj/item/clothing/suit/armor/ego_gear/innocence
+	name = "childhood memories"
+	desc = "In my dreams as child, Peter Pan would reach out a hand for me to hold and take me to Neverland. I had forgotten all of that, until I went into that room."
+	icon_state = "innocence"
+	armor = list(RED_DAMAGE = 10, WHITE_DAMAGE = 60, BLACK_DAMAGE = 60, PALE_DAMAGE = 10) //140
```
Comment (Kitsunemitsu) [inline code/modules/clothing/suits/ego_gear/waw.dm:350]: This might be too much

File: code/modules/projectiles/projectile/ego_bullets/waw.dm [hunk 2]
```diff
@@ -323,3 +323,10 @@
 	damage_type = PALE_DAMAGE
 	speed = 2
 	range = 6
+
+/obj/projectile/ego_bullet/ego_innocence
+	name = "innocence"
+	icon_state = "energy"
+	damage = 12 //Can dual wield, full auto
```
Comment (Kitsunemitsu) [inline code/modules/projectiles/projectile/ego_bullets/waw.dm:330]: Reduce to 6 or 7,  now it's on par with correctional, for one of them

File: code/modules/projectiles/projectile/ego_bullets/teth.dm [hunk 3]
```diff
@@ -55,3 +55,30 @@
 	damage = 20
 	damage_type = BLACK_DAMAGE
 	flag = BLACK_DAMAGE
+
+//Snapshot, hitscan laser
+/obj/projectile/beam/snapshot
+	name = "snapshot"
+	icon_state = "snapshot"
+	hitsound = null
+	damage = 16
```
Comment (Kitsunemitsu) [inline code/modules/projectiles/projectile/ego_bullets/teth.dm:64]: Bump up to 18

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]

File: code/modules/clothing/suits/ego_gear/waw.dm (hunk 1 above)
Comment (Coxswain-Navigator) [reply code/modules/clothing/suits/ego_gear/waw.dm:350]: What do you propose as an alternative here? 10 red and pale really is not actually very good

**Comment by lovegreenstuff** [comment]
it's just "lovegreenstuff" not "ILoveGreenstuff" there's no I in there
get it right nerd

**Review by Kitsunemitsu — APPROVED** [review]
Send me the stats on discord for the armor, Approving and merging.

If this is merged and is imbalanced we can change it as the TM goes out

**Pull request merged**